From ad2e4bf78e53307ed2d02c42e0fc2ed19d6118eb Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Sun, 23 Feb 2014 12:50:50 +0000
Subject: [PATCH 512/514] Revert "systemctl: skip native unit file handling if
 sysv file handling already handled everything"

This reverts commit 345151292fb48647cb9e9d5c6b4cd5dff0d0452a.

In Mageia 4, we have patches to chkconfig so we'll use this as our
solution for now rather than this patch.
---
 src/systemctl/systemctl.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/systemctl/systemctl.c b/src/systemctl/systemctl.c
index da49da7..bc9808a 100644
--- a/src/systemctl/systemctl.c
+++ b/src/systemctl/systemctl.c
@@ -4436,11 +4436,6 @@ static int enable_unit(DBusConnection *bus, char **args) {
         if (r < 0)
                 return r;
 
-        /* If the operation was fully executed by the SysV compat,
-         * let's finish early */
-        if (strv_isempty(mangled_names))
-                return 0;
-
         if (!bus || avoid_bus()) {
                 if (streq(verb, "enable")) {
                         r = unit_file_enable(arg_scope, arg_runtime, arg_root, mangled_names, arg_force, &changes, &n_changes);
-- 
1.8.4.5

